Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs

```diff
@@ -100,30 +105,36 @@ public async Task TrackEventWithPropertiesAsync()
             {
                 Assert.NotNull(typedProperties.SelectToken($"[?(@.name == 'Key{i}' && @.value == 'Value{i}')]"));
             }
-            Assert.Equal(1, httpNetworkAdapter.CallCount);
+            Assert.Equal(2, httpNetworkAdapter.CallCount);
         }
 
         [Fact]
         public async Task TrackEventNormalFlowAsync()
         {
-            var httpNetworkAdapter = new HttpNetworkAdapter(expectedLogType: "event");
+            var typeEvent = "event";
+            var httpNetworkAdapter = new HttpNetworkAdapter();
             DependencyConfiguration.HttpNetworkAdapter = httpNetworkAdapter;
+            var startServiceTask = httpNetworkAdapter.MockRequestByLogType("startService");
+            var eventTask = httpNetworkAdapter.MockRequestByLogType(typeEvent);
 
             // Start App Center.
             AppCenter.UnsetInstance();
             Analytics.UnsetInstance();
             AppCenter.LogLevel = LogLevel.Verbose;
             AppCenter.Start(Config.resolveAppsecret(), typeof(Analytics));
 
+            // Wait for "startService" log to be sent.
+            await startServiceTask;
+
             // Pause Analytics module.
             Analytics.Pause();
 
             // Test TrackEvent.
             Analytics.TrackEvent("TrackEvent 1");
 
             // Wait for 5 seconds to allow batching happen (3 seconds), and verify nothing has been sent.
-            Task.WaitAny(httpNetworkAdapter.HttpResponseTask, Task.Delay(5000));
-            Assert.Equal(0, httpNetworkAdapter.CallCount);
+            Task.WaitAny(eventTask, Task.Delay(5000));
+            Assert.Equal(1, httpNetworkAdapter.CallCount);
```

Comment:
If this is the global count you need to also assert 1 after startService log to clarify there was no increment since last time we checked. Otherwise it would look like a bug.